packages/test/src/FixtureManager.ts

```diff
@@ -26,10 +26,16 @@ export default class FixtureManager implements Manager {
       ): Promise<void> => {
         switch (action.type) {
           case actionTypes.FETCH_TYPE: {
-            const { key } = action.meta;
+            const { key, resolve, reject } = action.meta;
 
             if (key in this.mockResults) {
-              dispatch(this.mockResults[key] as any);
+              // All updates must be async or React will complain about re-rendering in same pass
+              setTimeout(() => {
+                const action: any = this.mockResults[key];
+                dispatch(action);
```

Comment:
this will still not reject if `dispatch` throws

Reply:
reducer will never throw due to try/catch. I guess if a user created manager throws this could cause suspense to never resolve. not sure we should send that error to the component tho. I'll put it in a finally to be sure